modules/primer-markdown/lib/code.scss

```diff
@@ -3,19 +3,11 @@
   // Inline code snippets
   code,
   tt {
-    padding: 0;
-    padding-top: 0.2em;
-    padding-bottom: 0.2em;
+    padding: 0.2em 0.4em;
```

Comment:
One day in the future we should bring this into our proposed [em-based scale](https://github.com/primer/primer-css/blob/master/modules/primer-support/docs/spacing.md#em-based-spacing) and stick to common fractions.